src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -266,14 +431,44 @@ public void missingSoundMapping(MissingMappingsEvent event)
     // done in an event instead of deferred to only enable if a mod requests it
     public void registerFluids(RegisterEvent event)
     {
-        if (enableMilkFluid && event.getRegistryKey().equals(ForgeRegistries.Keys.FLUIDS))
+        if (enableMilkFluid)
         {
-            // set up attributes
-            FluidAttributes.Builder attributesBuilder = FluidAttributes.builder(new ResourceLocation("forge", "block/milk_still"), new ResourceLocation("forge", "block/milk_flowing")).density(1024).viscosity(1024);
-            ForgeFlowingFluid.Properties properties = new ForgeFlowingFluid.Properties(MILK, FLOWING_MILK, attributesBuilder).bucket(() -> Items.MILK_BUCKET);
-            // register fluids
-            event.register(ForgeRegistries.Keys.FLUIDS, MILK.getId(), () -> new ForgeFlowingFluid.Source(properties));
-            event.register(ForgeRegistries.Keys.FLUIDS, FLOWING_MILK.getId(), () -> new ForgeFlowingFluid.Flowing(properties));
+            if (event.getRegistryKey().equals(ForgeRegistries.Keys.FLUID_TYPES))
```

Comment:
I feel like this check is unneeded as it is performed automatically by the event.register helper